jailer.go

```diff
@@ -431,13 +445,15 @@ func LinkFilesHandler(rootfs, kernelImageFileName string) Handler {
 type NaiveChrootStrategy struct {
 	Rootfs          string
 	KernelImagePath string
+	InitrdPath      string
 }
 
 // NewNaiveChrootStrategy returns a new NaivceChrootStrategy
-func NewNaiveChrootStrategy(rootfs, kernelImagePath string) NaiveChrootStrategy {
+func NewNaiveChrootStrategy(rootfs, kernelImagePath, initrdPath string) NaiveChrootStrategy {
```

Comment:
since this is an optional field, I would much prefer functional options here.

```go
type NaiveChrootStrategyOption func(*NaiveChrootStrategy)

func NewNaiveChrootStrategy(rootfs, kernelImagePath, options ...NaiveChrootStrategyOption) NaiveChrootStrategy
```